### PR TITLE
fix(website): Add running locally

### PR DESCRIPTION
**Pull request by disq**
Implements #2868

### CODE REVIEW

File: website/pages/docs/developers/running-locally.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/path/to/plugin serve
+```
+
+If you are running multiple plugins this way simultaneously, you will need to specify a different port for each one. You can do this with the `--address` flag:
+
+```bash
+/path/to/plugin serve --address localhost:7778
+```
+
+> 1:16PM INF Source plugin server listening address=127.0.0.1:7778
```
Comment (erezrokah) [inline website/pages/docs/developers/running-locally.md:41]:
```suggestion
> 1:16 PM INF Source plugin server listening address=127.0.0.1:7778
```

File: website/pages/docs/developers/running-locally.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+/path/to/plugin serve
+```
+
+If you are running multiple plugins this way simultaneously, you will need to specify a different port for each one. You can do this with the `--address` flag:
+
+```bash
+/path/to/plugin serve --address localhost:7778
+```
+
+> 1:16 PM INF Source plugin server listening address=127.0.0.1:7778
```
Comment (erezrokah) [inline website/pages/docs/developers/running-locally.md:41]:
```suggestion
> `1:16PM INF Source plugin server listening address=127.0.0.1:7778`
```

File: website/pages/docs/developers/running-locally.md (hunk 1 above)
Comment (disq) [reply website/pages/docs/developers/running-locally.md]: But that's not the logline says 😂 Happy to commit that to make Vale happy tho

Comment (erezrokah) [reply website/pages/docs/developers/running-locally.md]: Added another commit to put it in a code block, but also maybe fix the log line (probably not needed)

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**